src/test/java/salesforce/opportunity/OpportunityTests.java

```diff
@@ -0,0 +1,107 @@
+package salesforce.opportunity;
+
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import salesforce.base.BaseTest;
+import salesforce.ui.pages.opportunity.CreatedOpportunity;
+import salesforce.ui.pages.opportunity.NewOpportunityPage;
+import salesforce.ui.pages.opportunity.OpportunityPage;
+
+public class OpportunityTests extends BaseTest {
+
+    private NewOpportunityPage formOpportunity;
+    private CreatedOpportunity createdForm;
+    private OpportunityPage opportunityPageBack;
+    SoftAssert softAssert = new SoftAssert();
+
+    @Test
+    public void testCreateOpportunityWithRequiredValuesOnly() {
+        String opportunityName = "New opportunity name";
+        String opportunityCloseDate = "7/31/2021";
+        String opportunityStage = "Prospecting";
+
+        formOpportunity = opportunityPage.openNewOpportunityForm();
+        formOpportunity.setDropdown("Stage", opportunityStage);
+        formOpportunity.setInputField("Name", opportunityName);
+        formOpportunity.setCloseDate(opportunityCloseDate);
+        createdForm = formOpportunity.clickSaveOpportunity();
+        //Assert with Alert when creating
```

Comment:
this comments should be removed

Reply:
These comments are just there to sepárele the different asserts, but I will remove it in the next PR